builders/interact/ethereum-api/libraries/web3js.md

```diff
@@ -0,0 +1,498 @@
+---
+title: Send Transactions & Deploy Contracts with Web3.js
+description: Follow this tutorial to learn how to use the Ethereum Web3 JavaScript Library to send transactions and deploy Solidity smart contracts to your Tanssi EVM ContainerChain.
+---
+
+# Web3.js JavaScript Library
+
+## Introduction {: #introduction }
+
+[Web3.js](https://web3js.readthedocs.io/){target=_blank} is a set of libraries that allow developers to interact with Ethereum nodes using HTTP, IPC, or WebSocket protocols with JavaScript. Tanssi EVM ContainerChains have an Ethereum-like API available that is fully compatible with Ethereum-style JSON RPC invocations.  Therefore, developers can leverage this compatibility and use the Web3.js library to interact with a Tanssi EVM ContainerChain node as if they were doing so on Ethereum. For more information on Web3.js, check their [documentation site](https://web3js.readthedocs.io/en/v1.10.0/){target=_blank}.
+
+In this guide, you'll learn how to use setup the Web3.js library for your Tanssi EVM ContainerChain. Next, to showcase the library in action, you'll use Web3.js library to send a transaction and deploy a contract on a Tanssi EVM ContainerChain running in Tanssi's [Dancebox](XXX){target=_blank} TestNet. This guide can be adapted for your own Tanssi EVM ContainerChain by simply changing the endpoint.
+
+--8<-- 'text/common/general-js-tutorial-check.md'
+
+## Checking Prerequisites {: #checking-prerequisites } 
+
+For the examples in this guide, you will need to have the following:
+
+ - An account with funds in the Tanssi EVM ContainerChain you are testing with
+
+## Create a JavaScript Project {: #create-a-javascript-project }
+
+To get started, you can create a directory to store all of the files you'll be creating throughout this guide:
```

Comment:
I believe this changed in the moonbeam-docs repo to include initializing the project. Please update this section to match

ref: https://github.com/moonbeam-foundation/moonbeam-docs/blob/master/builders/build/eth-api/libraries/web3js.md?plain=1#L28